Source/WTF/Scripts/Preferences/WebPreferences.yaml

```diff
@@ -276,31 +276,26 @@ AppleMailPaginationQuirkEnabled:
     WebCore:
       default: false
 
-# FIXME: Is this implemented for WebKitLegacy? If not, this should be excluded from WebKitLegacy entirely (though we should still set the default value to false when initializing settings).
```

Comment:
I don't understand why you removed the fixme if you didn't fix what it is complaining about? (with an `exposed` annotation)

Reply:
oh i maybe misunderstood this.

i tried removing the `WebKitLegacy:`, but the generator script complained about it having to be defined for all three if it's defined for `WebCore:`

TIL about `exposed:`!  i'm not really sure what that does exactly, but basically we only want this to be turned on when using WK2 as WK1/WebCore do not have the necessary platform hooks to actually use Apple Pay (e.g. PassKit).  is `exposed` something i should be using to achieve that?

Reply:
Right! I forgot that you can't hide it from WKL *only*. So, I think your patch is fine.